From 0198cdd559c639ebd8cb6badfaefedb06588b13b Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 13 Jan 2022 14:53:43 +0100
Subject: [PATCH] mariadb: enable it by default

We need to have it enabled by default because of several reasons:
a) Nextcloud uses it
b) We need to migrate from Turris OS 5.x , where the mariadb was
preconfigured
---
 utils/mariadb/files/mysqld.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/mariadb/files/mysqld.config b/utils/mariadb/files/mysqld.config
index e33adefc8..4889c0970 100644
--- a/utils/mariadb/files/mysqld.config
+++ b/utils/mariadb/files/mysqld.config
@@ -1,7 +1,7 @@
 
 config mysqld 'general'
     # Unless enable, MariaDB will not start without this
-	option enabled '0'
+	option enabled '1'
     # User to run MariaDB as
     option user 'mariadb'
     # Group to run MariaDB
-- 
2.32.0

